src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -43,11 +43,6 @@ partial void UpdatePlatform()
 			var newState = new BorderLayerState(drawArea.Size, _borderInfoProvider);
 			var previousLayoutState = _currentState;
 
-			if (newState.Equals(previousLayoutState))
-			{
-				return;
-			}
```

Comment:
This looks like a performance issue. Do you recall why you needed to remove it?

Reply:
I managed to create a repro that's completely independent of GLCanvasElement to illustrate the problem and opened a separate PR.
https://github.com/unoplatform/uno/pull/18408